test.sh.in

```diff
@@ -252,12 +252,33 @@ nodesCreate() {
   fi
 }
 
+vagrantSuspendAll() {
+ for box in `vagrant global-status --prune | awk '/running/{print $1}'`; do
+     vagrant suspend ${box}
```

Comment:
I agree your statement about "a slave server one purpose at a time, so no other vagrant instance should run"